Gordon360/Controllers/LostAndFoundController.cs

```diff
@@ -36,6 +36,21 @@ public async Task<ActionResult> UpdateMissingItemReport(int id, [FromBody] Missi
             return Ok();
         }
 
+        /// <summary>
+        /// Update Item Report Status to "Deleted" with the given id
+        /// </summary>
+        /// <param name="id">The id</param>
+        /// <returns></returns>
+        [HttpPut]
+        [Route("missingitem/status")]
```

Comment:
Also, you'll need to specify the id in the route here.  I also thing that variables in the route need to be enclosed in {}.  So something like missingitem/{id}/{status} I believe will work